### PR TITLE
Wso 62 html cleanup

### PR DESCRIPTION
**Pull request by SandyVallee**

### CODE REVIEW

File: sites/all/modules/ucalgary/ucgy_gbl/ucgy_gbl_cta/templates/fieldable-panels-pane--gbl-hero-banner.tpl.php [hunk 1]
```diff
@@ -81,14 +81,15 @@
                         <img id="herobg<?php echo $id; ?>" class="img-responsive" src="<?php echo $hero_img; ?>">
                     <?php endif; ?>
                 </div>
-                <?php if(!empty($content['title']['#value']) || isset($content['field_cta_description']) || isset($content['field_cta_buttons'])) { ?>
+                <?php if(!empty($content['title']['#value']) || !empty($content['field_cta_description']) || !empty($content['field_cta_buttons'])) { ?>
                     <div class="col-sm-12">
                         <div class="cta-content <?php echo $text_align;?>">
-                            <?php if (isset($content['title']['#value'])): ?>
+                            <?php if (!empty($content['title']['#value'])): ?>
```
Comment (christophergraha1) [inline sites/all/modules/ucalgary/ucgy_gbl/ucgy_gbl_cta/templates/fieldable-panels-pane--gbl-hero-banner.tpl.php]: Please change your if statement to standard format: if(){}

File: sites/all/modules/ucalgary/ucgy_gbl/ucgy_gbl_cta/templates/fieldable-panels-pane--gbl-hero-banner.tpl.php [hunk 2]
```diff
@@ -81,14 +81,15 @@
...
                                 <h1 class="<?php echo $hide_title ?>"><?php echo $content['title']['#value']; ?></h1>
                             <?php endif; ?>
-                            <p><?php echo render($content['field_cta_description']); ?></p>
-
+                            <?php if (!empty($content['field_cta_description'])): ?>
```
Comment (christophergraha1) [inline sites/all/modules/ucalgary/ucgy_gbl/ucgy_gbl_cta/templates/fieldable-panels-pane--gbl-hero-banner.tpl.php]: Please change your if statement to standard format: if(){}

File: sites/all/modules/ucalgary/ucgy_gbl/ucgy_gbl_cta/templates/fieldable-panels-pane--gbl-hero-banner.tpl.php [hunk 3]
```diff
@@ -81,14 +81,15 @@
                         <img id="herobg<?php echo $id; ?>" class="img-responsive" src="<?php echo $hero_img; ?>">
```
Comment (christophergraha1) [inline sites/all/modules/ucalgary/ucgy_gbl/ucgy_gbl_cta/templates/fieldable-panels-pane--gbl-hero-banner.tpl.php]: Please change your if statement to standard format: if(){}

File: sites/all/modules/ucalgary/ucgy_gbl/ucgy_gbl_cta/templates/fieldable-panels-pane--gbl-hero-banner.tpl.php [hunk 4]
```diff
@@ -81,14 +81,15 @@
...
                     <?php endif; ?>
                 </div>
-                <?php if(!empty($content['title']['#value']) || isset($content['field_cta_description']) || isset($content['field_cta_buttons'])) { ?>
+                <?php if(!empty($content['title']['#value']) || !empty($content['field_cta_description']) || !empty($content['field_cta_buttons'])) { ?>
                     <div class="col-sm-12">
                         <div class="cta-content <?php echo $text_align;?>">
```
Comment (christophergraha1) [inline sites/all/modules/ucalgary/ucgy_gbl/ucgy_gbl_cta/templates/fieldable-panels-pane--gbl-hero-banner.tpl.php]: Space out end php tag away from content

File: sites/all/modules/ucalgary/ucgy_gbl/ucgy_gbl_cta/templates/fieldable-panels-pane--gbl-hero-banner.tpl.php (hunk 2 above)
Comment (christophergraha1) [inline sites/all/modules/ucalgary/ucgy_gbl/ucgy_gbl_cta/templates/fieldable-panels-pane--gbl-hero-banner.tpl.php]: Remove space from initial if, example should be if() not if ()

File: sites/all/modules/ucalgary/ucgy_gbl/ucgy_gbl_cta/templates/fieldable-panels-pane--gbl-hero-banner.tpl.php (hunk 1 above)
Comment (SandyVallee) [inline sites/all/modules/ucalgary/ucgy_gbl/ucgy_gbl_cta/templates/fieldable-panels-pane--gbl-hero-banner.tpl.php]: done

File: sites/all/modules/ucalgary/ucgy_gbl/ucgy_gbl_cta/templates/fieldable-panels-pane--gbl-hero-banner.tpl.php (hunk 2 above)
Comment (SandyVallee) [inline sites/all/modules/ucalgary/ucgy_gbl/ucgy_gbl_cta/templates/fieldable-panels-pane--gbl-hero-banner.tpl.php]: done

File: sites/all/modules/ucalgary/ucgy_gbl/ucgy_gbl_cta/templates/fieldable-panels-pane--gbl-hero-banner.tpl.php (hunk 3 above)
Comment (SandyVallee) [inline sites/all/modules/ucalgary/ucgy_gbl/ucgy_gbl_cta/templates/fieldable-panels-pane--gbl-hero-banner.tpl.php]: done

File: sites/all/modules/ucalgary/ucgy_gbl/ucgy_gbl_cta/templates/fieldable-panels-pane--gbl-hero-banner.tpl.php (hunk 4 above)
Comment (SandyVallee) [inline sites/all/modules/ucalgary/ucgy_gbl/ucgy_gbl_cta/templates/fieldable-panels-pane--gbl-hero-banner.tpl.php]: done

File: sites/all/modules/ucalgary/ucgy_gbl/ucgy_gbl_cta/templates/fieldable-panels-pane--gbl-hero-banner.tpl.php (hunk 2 above)
Comment (SandyVallee) [inline sites/all/modules/ucalgary/ucgy_gbl/ucgy_gbl_cta/templates/fieldable-panels-pane--gbl-hero-banner.tpl.php]: decided to go with spaces as per our conversation with Dan and Chris and Jesse

**Pull request merged**